File: cpp/combustivel.cpp
#include <iostream>

using namespace std;

int main(){
    int codigo, alcool = 0, gasolina = 0, diesel = 0;

    cout << "Informe um codigo (1, 2, 3) ou 4 para parar: ";
    cin >> codigo;

    while(codigo!=4){
        if(codigo == 1){
            alcool++;
        }
        else if(codigo == 2){
            gasolina++;
        }
        else if(codigo == 3){
            diesel++;
        }
        cout << "Informe um codigo (1, 2, 3) ou 4 para parar: ";
        cin >> codigo;
    }

    cout << "MUITO OBRIGADO" << endl;
    cout << "Alcool: " << alcool << endl;
    cout << "Gasolina: " << gasolina << endl;
    cout << "Diesel: " << diesel << endl;

	return 0;
}

File: python/combustivel.py
codigo: int; alcool: int; gasolina: int; diesel: int

codigo = int(input("Informe um codigo (1, 2, 3) ou 4 para parar: "))

alcool = 0
gasolina = 0
diesel = 0

while codigo != 4:
	if codigo == 1:
		alcool = alcool + 1
	elif codigo == 2:
		gasolina = gasolina + 1
	elif codigo == 3:
		diesel = diesel + 1
	codigo = int(input("Informe um codigo (1, 2, 3) ou 4 para parar: "))

print("MUITO OBRIGADO")
print(f"Alcool: {alcool}")
print(f"Gasolina: {gasolina}")
print(f"Diesel: {diesel}")